cpp/oneapi/dal/backend/primitives/selection/kselect_by_rows_quick.hpp

```diff
@@ -156,8 +162,9 @@ class kselect_by_rows_quick : public kselect_by_rows_base<Float> {
         if (row_id >= num_rows)
             return;
 
-        const std::int32_t offset_in = row_id * BlockOffset;
-        const std::int32_t offset_out = row_id * k;
+        const std::int32_t offset_in = row_id * inp_stride;
+        [[maybe_unused]] const std::int32_t offset_ids_out = row_id * out_ids_stride;
```

Comment:
is [[maybe_unused]] still needed?

Reply:
Yes. It is r3equired because of the constexpr if below